### PR TITLE
refactor(all): remove PBSOrder generic marker

### PR DESCRIPTION
**Pull request by tmontaigu**
The motivation for this change is that having the Big/Small being a generic parameter of types, makes the code more complex than it should be.

Also, It was not complete, not all structs had this generic parameter making the whole thing a bit more clunky.

### CODE REVIEW

**Review by twiby — APPROVED** [review]
Seems good to me, though of course this is really big, and something might have slipped me. I tried to scan everything (except the c/wasm/js bindings, because I don't know the code well enough), and it looks alright.
The change looks great, it's gonna be much simpler for users, and future maintainers as well.

File: tfhe/docs/js_on_wasm_api/tutorial.md
Comment (tmontaigu) [inline tfhe/docs/js_on_wasm_api/tutorial.md:1]: These should still be BigInt

**Comment by tmontaigu** [comment]
@slab-ci cpu_test
@slab-ci cpu_integer_test
@slab-ci cpu_multi_bit_test
@slab-ci cpu_wasm_test

File: tfhe/js_on_wasm_tests/test.js [hunk 1]
```diff
@@ -1,233 +1,233 @@
-const crypto = require('crypto');
```
Comment (IceTDrinker) [inline tfhe/js_on_wasm_tests/test.js:1]: missed that 🤦‍♂️

**Pull request merged**